resources/js/views/settings/Application.vue

```diff
@@ -345,6 +627,21 @@ export default {
       formData.append('room_limit', this.settings.room_limit);
       formData.append('pagination_page_size', this.settings.pagination_page_size);
       formData.append('own_rooms_pagination_page_size', this.settings.own_rooms_pagination_page_size);
+
+      Object.keys(this.settings.banner).forEach(key => {
+        let val = this.settings.banner[key];
+
+        // Since the FormData always strings boolean and empty values must be
```

Comment:
are you sure about that? https://laravel.com/docs/8.x/validation#rule-boolean

Reply:
Yes! Due to the FormData, booleans sent as `'true'` and `'false'` and null values as `'null'`. This is not correct since the nullable Rule in the validation wouldn't work and for the boolean validation also will not work since `'true'` != `true`. Only `'0'` and `'1'` supported as strings!